### ISSUE TITLE
Parse / ignore quotes in .cnf values

### ISSUE THREAD

**Issue by makmanalp** [sjmudd/mysql_defaults_file#1]
Hi,

Thanks for the very neat `ps-top` tool - it's incredibly helpful!

We had some trouble with it with our preexisting `~/.my.cnf` login files. We figured out that the reason is that we quote some field values, eg `password="blah"`. We do this because auto-generated passwords can contain special characters like `#;\` all of which have meaning in the my.cnf spec - e.g. a `#` in a password string would be interpreted by the mysql CLI tools as a comment. The quotation marks allow you to express a literal. It seems the my.cnf and .ini formats are similar but [not really compatible](https://dev.mysql.com/doc/refman/5.7/en/option-files.html).

On the positive side, it looks like the ini parser you use pretty much isn't doing any interpreting (i.e. every config value currently is a literal, `#;` etc that's not at the beginning of a line is [ignored anyway](https://github.com/vaughan0/go-ini/blob/a98ad7ee00ec53921f08832bc06ecf7fd600e6a1/ini.go#L89)), so perhaps just trimming surrounding quotes from the .ini values actually "fixes" this problem for now, until - if ever - you decide to support those other constructs.

eg

```golang
func quoteTrim(val string) string {
    valLen = len(val)
    if valLen >= 2 {
        if val[0] == '"' && val[valLen-1] == '"' {
            return val[1 : valLen-1]
        }
    }
    return val
}
```

and then use that all around [here](https://github.com/sjmudd/mysql_defaults_file/blob/2fb1aa878dfea1576b7f7b53684b0b55c3080261/mysql_defaults_file.go#L37).

How does this sound? I'm happy to give this a go and make a PR if that's acceptable to you.

**Comment by sjmudd** [comment]
Hi.  Thanks for the comment. I've never bumped into this before. The suggestion you've made sounds reasonable. It would be ideal if the upstream package handled this and I just used it so I'd suggest you perhaps provide a feature request there.

In the meantime I'll look to add a patch to handle this.

**Comment by sjmudd** [comment]
@makmanalp I've also trimmed whitespace as that seems sensible first. Could you check if my change works for you?
If it does I'll merge it and apply to ps-top.

**Comment by sjmudd** [comment]
I added some checks which looked good, and this has been merged.
Also incorporated these changes into the latest ps-top at https://github.com/sjmudd/ps-top/releases/tag/v0.7.15.

**Comment by makmanalp** [comment]
Sorry for the delay in getting back to you, that was super quick - thank you very much for that. I just tested it out and it works!

One thing to note is that in theory any field's value can be quoted, eg we had the username quoted for some reason (probably unnecessary) too and that still trips ps-top up, but I've changed stuff to deal with that on our end - up to you to decide it's worth applying it to all the fields or not.
